apps/passport-client/src/useSyncE2EEStorage.tsx

```diff
@@ -107,6 +112,19 @@ export async function uploadStorage(
   pcds: PCDCollection,
   subscriptions: FeedSubscriptionManager
 ): Promise<UploadStorageResult> {
+  const validationErrors = validateUpload(user, pcds);
+  if (validationErrors.length > 0) {
+    logAndUploadValidationErrors(validationErrors);
+    return {
```

Comment:
User should probably get an Invalid Passport dialog to force them to log out at this point.